@wumpus I added back the checks you pointed in the review comment, I updated the instruction to point users to cc-downloader. 

Outside the common crawl foundation: 

```
(cc) ➜  whirlwind-python git:(feature/luca/update-duckdb) ✗ make duck_local_files LOCAL_DIR=~/Downloads
warning! 300 gigabyte download
python duck.py local_files ~/Downloads
Using local prefix /Users/lfoppiano/Downloads
Do you need to download this index?
The recommended way is to use cc-downloader https://github.com/commoncrawl/cc-downloader
If you have cargo and Rust already installed: `cargo install cc-downloader` (alternatively, the binaries are available on the GitHub repository) , and then, 
  ~/.cargo/bin/cc-downloader download-paths CC-MAIN-2024-22 cc-index-table /Users/lfoppiano/Downloads
  ~/.cargo/bin/cc-downloader download /Users/lfoppiano/Downloads/cc-index-table.paths.gz --progress /Users/lfoppiano/Downloads
make: *** [duck_local_files] Error 1
```

Inside the common crawl foundation: 
```
(cc) ➜  whirlwind-python git:(feature/luca/update-duckdb) ✗ make duck_ccf_local_files                  
warning! only works on Common Crawl Foundation's development machine
python duck.py ccf_local_files
Do you need to download this index?
The recommended way is to use cc-downloader https://github.com/commoncrawl/cc-downloader
If you have cargo and Rust already installed: `cargo install cc-downloader` (alternatively, the binaries are available on the GitHub repository) , and then, 
  ~/.cargo/bin/cc-downloader download-paths CC-MAIN-2024-22 cc-index-table /home/cc-pds/
  ~/.cargo/bin/cc-downloader download /home/cc-pds//cc-index-table.paths.gz --progress /home/cc-pds/
make: *** [duck_ccf_local_files] Error 1
```

--- a/duck.py
+++ b/duck.py
@@ -68,7 +68,7 @@ def get_files(algo, crawl, local_prefix=None):
         # Check whether the local files have been already downloaded
         # We expect 300 files of about a gigabyte
         if len(files) < 250:
-            index_download_advice('/home/cc-pds/', crawl)
+            index_download_advice('/home/cc-pds', crawl)
             exit(1)
     elif algo == 'cloudfront_glob':
         # duckdb can't glob this, same reason as s3_glob above

